index.html

```diff
@@ -0,0 +1,264 @@
+              <li>
+                A platform that supports user uploaded videos like Tiktok or
+                Instagram
+              </li>
+              <li>Video into a custom CMS</li>
+              <li>And more ...</li>
+            </ul>
+            <p>
+              To learn more, read the tutorial or
+              <a href="/">check out the source code on GitHub.</a>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <script>
+      const sidebar = document.getElementById('sidebar');
+      const content = document.getElementById('content');
+      const chevronRight = document.getElementById('chevron-right');
+      const chevronLeft = document.getElementById('chevron-left');
+      const fullLogo = document.getElementById('full-logo');
+      const miniLogo = document.getElementById('mini-logo');
+      const uploadContainer = document.querySelector('.upload-container');
+      const dropZone = document.querySelector('.dnd-container');
+      const dndDesktop = document.querySelector('.dnd-desktop');
+      const dndMobile = document.querySelector('.dnd-mobile');
+      const right = document.getElementById('right');
+
+      window.onload = function() {
+        dropZone.addEventListener('dragover', function(event) {
+          event.preventDefault();
+        })
+        dropZone.addEventListener('drop', function(event) {
+          event.preventDefault();
+          if (event.dataTransfer.items) {
+            [...event.dataTransfer.items].forEach((item, i) => {
+              if (item.kind === 'file') {
+                const file = item.getAsFile();
+                uploadFile(file);
+              }
+            });
+          } else {
+            [...event.dataTransfer.files].forEach((file, i) => {
+              uploadFile(file);
+            });
+          }
+        })
+      }
+
+      function closeNav() {
+        sidebar.style.width = '20';
+        content.classList.add('hide');
+        chevronRight.classList.remove('hide');
+        chevronLeft.classList.add('hide');
+        fullLogo.classList.add('hide');
+        miniLogo.classList.remove('hide');
+        right.style.width = '100%';
+      }
+
+      function openNav() {
+        sidebar.style.width = '400';
+        content.classList.remove('hide');
+        fullLogo.classList.remove('hide');
+        chevronRight.classList.add('hide');
+        chevronLeft.classList.remove('hide');
+        right.style.removeProperty('width');
+      }
+
+      function openFilePicker() {
+        document.getElementById('file-input').click();
+      }
+
+      function fileInputChange(e) {
+        const { files } = e;
+        if (files && files.length > 0) {
+          uploadFile(files[0]);
+        }
+      }
+
+      function uploadFile(file) {
+        const uploader = new VideoUploader({
+          file,
+          uploadToken: 'YOUR_UPLOAD_TOKEN',
+        });
+        uploader.upload();
```

Comment:
I would catch potential errors here. For instance if the delegated upload token is wrong it fails silently for now
```
uploader.upload().catch((e) => {
   // something simple that let the user know that there was an error
   // simply: alert(e.title); ?
});
```